### ISSUE TITLE
"This app has been blocked by your system administrator"

### ISSUE THREAD

**Issue by junglejuz** [ChrisTitusTech/win10script#72]
Getting this message when trying to launch "Your Phone" in Windows 10. Some other software also won't run at all now either.

**Comment by rcabecas** [comment]
I also have this issue. Can't make "My Phone" app work no matter what.

**Comment by mikhoul** [comment]
In meantime: https://www.windowsbasics.com/2020/06/this-app-has-been-blocked-by-your.html

**Comment by jcharbo94** [comment]
I managed to resolve the issue with the Your Phone app by editing 3 registry key and a few settings:
Open regedit and change those keys:
- Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\System\EnableMmx  set to 1
- Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\System\EnableCdp set to 1
- Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\Messaging\AllowMessageSync set to 1 (not sure this one is necessary tho)

Reboot

Then, in Windows 10 settings app:
- System --> Shared Experience: Enable"Share across devices" and make sure "All accounts are working correctly"
- Privacy --> Background apps: Enable "Background App" and make sure it is enabled for Your phone app
- Apps --> Your Phone --> Advanced Options --> Terminate and Repair

I hope this helps

**Comment by rcabecas** [comment]
This **WORKED**!
Thank you so much @jcharbo94 !! 

> I managed to resolve the issue with the Your Phone app by editing 3 registry key and a few settings:
> Open regedit and change those keys:
> 
> * Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\System\EnableMmx  set to 1
> * Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\System\EnableCdp set to 1
> * Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\Messaging\AllowMessageSync set to 1 (not sure this one is necessary tho)
> 
> Reboot
> 
> Then, in Windows 10 settings app:
> 
> * System --> Shared Experience: Enable"Share across devices" and make sure "All accounts are working correctly"
> * Privacy --> Background apps: Enable "Background App" and make sure it is enabled for Your phone app
> * Apps --> Your Phone --> Advanced Options --> Terminate and Repair
> 
> I hope this helps

**Comment by skynet-05** [comment]
Nope, none of the above worked I'm not sure why. If there's anything else, it'd be helpful.

**Comment by ChrisTitusTech** [comment]
Adding Fix for Your Phone in next commit. It should do everything detailed by jcharbo
